src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
@mobilunity-user But when using TS typings users would need to deal with `IHash` alias:
- see IHash, be like "what's that"
- go to definition
- "Oh it's just an object with any properties, nevermind"
- pass object
rather than go ahead and pass `object` in the first place.